tests/test_coastal_blue_carbon.py

```diff
@@ -358,7 +358,6 @@ def test_read_transition_matrix(self):
             transition_csv.write('b,,NCC,accum\n')
             transition_csv.write('c,accum,,NCC\n')
             transition_csv.write(',,,\n')
-            transition_csv.write(',legend,,')  # simulate legend
```

Comment:
Here's one minor functional change. This was testing that extra stuff in the table would be ignored. I don't think we allow that in other tables in invest, so it seemed easier to drop it from the test than figure out how to accomplish that with `read_csv_to_dataframe`. But let me know if there's a reason to keep it!

Reply:
Yep, you're right that this table legend is unique among the InVEST models!  It's also weird in that it breaks from the normal table structure only to provide some extra metadata that's also described in the UG chapter.  Personally I'm OK with removing it, but because this is a notable functional change we should also be sure to 

- [ ] remove the legend-generating code from the preprocessor
- [ ] Note the functional change in the model's HISTORY
- [ ] remove the legend from the `transitions_template.csv` in the sample data repo

What do you think?

Reply:
Ah, I didn't realize that the preprocessor and sample data relied on this. In the interest of self-contained PRs, I'd rather not have to make all those changes here.

How about this for now? I made a small change in `coastal_blue_carbon.py` so that it will skip over rows that have nothing in the `lulc-class` column (which should be all the legend rows). That way the functionality is the same, but `read_csv_to_dataframe` doesn't have to handle it. And we can remove the legend in a separate issue if it's worth it?

Reply:
Sounds like a fine solution to me!  I took the liberty of describing this in https://github.com/natcap/invest/issues/1336